### ISSUE TITLE
Loss function returns NaN

### ISSUE THREAD

**Issue by maxim0815** [isl-org/Open3D-ML#441]
Hey guys, thanks for sharing and supporting your code. Extending Open3d with ML-tools is quite handy. 

## Description

There seems to be a problem with the SemantiSegmentation loss (torch). 

Training SparseConvNet on S3DIS dataset returns NaN within the train and eval loss:
![image](https://user-images.githubusercontent.com/57002046/145815649-50396ac1-ae77-4e5d-8a35-69431501aaa8.png)

Training PointTransformer on S3DIS dataset returns NaN within the eval loss:
![image](https://user-images.githubusercontent.com/57002046/145816547-b206e20d-30fc-4dce-846c-81b5fb42e8aa.png)

## Environment

- Docker environment (nvidia/cuda:11.0.3-cudnn8-devel-ubuntu20.04)
- Open3d [from source](http://www.open3d.org/docs/release/compilation.html)
- torch 1.8.2 [link](https://github.com/isl-org/open3d_downloads/releases/tag/torch1.8.2)
- python3.8

**Comment by sanskar107** [comment]
@maxim0815 I've found out that there is a NaN value in one of the file in S3DIS dataset. You can add this line in `ml3d/datasets/s3dis.py` to fix this.
https://github.com/isl-org/Open3D-ML/blob/sanskar/randlanet/ml3d/datasets/s3dis.py#L285

**Comment by Arno-Tu** [comment]
> There seems to be a problem with the SemantiSegmentation loss (torch).

Hello, I noticed that the link you gave is no longer working, can you tell me exactly which line of code is causing this problem? Thank you.

**Comment by Arno-Tu** [comment]
> @maxim0815 I've found out that there is a NaN value in one of the file in S3DIS dataset. You can add this line in `ml3d/datasets/s3dis.py` to fix this. https://github.com/isl-org/Open3D-ML/blob/sanskar/randlanet/ml3d/datasets/s3dis.py#L285

Hello, I noticed that the link you gave is no longer working, can you tell me exactly which line of code is causing this problem? Thank you.
